Doc/includes/sqlite3/converter_point.py

```diff
@@ -5,40 +5,33 @@ def __init__(self, x, y):
         self.x, self.y = x, y
 
     def __repr__(self):
-        return "(%f;%f)" % (self.x, self.y)
+        return f"Point({self.x}, {self.y})"
 
 def adapt_point(point):
-    return ("%f;%f" % (point.x, point.y)).encode('ascii')
+    return ("%f;%f" % (point.x, point.y)).encode("utf-8")
```

Comment:
Any reason not to use an f-string here as well?

```suggestion
    return f"{point.x};{point.y}".encode("utf-8")
```

Reply:
Thanks!